sdk/storage/AzureStorageBlob/Source/DataStructures/StorageBlobClientOptions.swift

```diff
@@ -48,11 +52,23 @@ public struct StorageBlobClientOptions: AzureConfigurable {
     public init(
         apiVersion: StorageBlobClient.ApiVersion = .latest,
         logger: ClientLogger = ClientLoggers.default(tag: "StorageBlobClient"),
-        maxChunkSizeInBytes: Int = 4 * 1024 * 1024 - 1
+        maxChunkSizeInBytes: Int = 4 * 1024 * 1024 - 1,
+        downloadNetworkPolicy: TransferNetworkPolicy? = nil,
+        uploadNetworkPolicy: TransferNetworkPolicy? = nil
     ) {
         self.apiVersion = apiVersion.rawValue
         self.logger = logger
         self.maxChunkSizeInBytes = maxChunkSizeInBytes
+
+        // apply default `TransferNetworkPolicy` if none supplied
+        self.downloadNetworkPolicy = downloadNetworkPolicy ?? TransferNetworkPolicy(
```

Comment:
May be a good idea to DRY this up into a `TransferNetworkPolicy.always` (naming?) or something? It could be internal if we don't want to expose it to the customer (I'm undecided on whether that would be a good idea or not).

Reply:
I made it an internal static variable named `default`.